src/Files/DTO/File.php

```diff
@@ -285,6 +286,20 @@ public function isText(): bool
         return $this->mimeType->isText();
     }
 
+    /**
+     * Checks if the file is a specific MIME type.
+     *
+     * @since n.e.x.t
+     *
+     * @param string $type The type to check.
+     *
+     * @return bool True if the file is of the specified type.
+     */
+    public function isType(string $type): bool
```

Comment:
This should be called `isMimeType`, since `isType` is not clear in what it means - there are two concepts of "type" here, one being "file type" and the other being "MIME type".